doc/hardware/cc2520.rst

```diff
@@ -0,0 +1,84 @@
+.. _cc2520:
+
+CC2520 setup
+##################
+
+Category: radio
+Standard: IEEE 802.15.4
+Boards: MinnowBoard MAX
+
+.. image:: _static/cc2520.png
```

Comment:
I've got some formatting suggestions here, such as adding attributes to control the size and positioning of the image, such as:

```
.. image:: _static/cc2520.png
   :width: 350px
   :alt: CC2520 board 
   :align: right
```

but I can do this kind of layout tweaking after this gets merged and I do an editing pass.